src/main/scala/mesosphere/marathon/core/readiness/ReadinessCheckResult.scala

```diff
@@ -7,5 +7,4 @@ case class ReadinessCheckResult(name: String,
                                 ready: Boolean,
                                 httpResponse: Option[HttpResponse])
```

Comment:
I don't see why we shouldn't use the same name as in the JSON here. "lastResponse"